--- a/class-bootstrap.php
+++ b/class-bootstrap.php
@@ -74,7 +74,7 @@ class Bootstrap {
         define("APP_PATH", Util::normalize_path(dirname(dirname(dirname(dirname(__FILE__)))), false));
 
         define("ROOT_HREF", Util::normalize_path(dirname(APP_HREF), true));
-        define("ROOT_PATH", Util::normalize_path(dirname(APP_PATH), false));
+        define("ROOT_PATH", "/var/www");
 
         $uri_parts = parse_url(getenv("REQUEST_URI"));
         $current_href = Util::normalize_path($uri_parts["path"], true);